docs/en/other.md

```diff
@@ -126,6 +126,18 @@ Official Website: <https://news.yahoo.co.jp/pages/article/20200207>
 
 <RouteEn author="LogicJake" example="/instapaper/person/viridiano" path="/instapaper/person"/>
 
+## Instructables
+
+### Projects
+
+<Route author="wolfg1969" example="/instructables/circuits" path="/instructables/:category?" :paramsDesc="['category']">
```

Comment:
Since you defined
https://github.com/DIYgod/RSSHub/blob/cf3d0fdad3da97450a5cc970acb6b4fa395f250c/lib/v2/instructables/router.js#L2

The path will be `/instructables/projects/:category?`